sabaki/README.md

```diff
@@ -0,0 +1,81 @@
+  ![bot-config-screenshot](bot-config-screenshot.png)
+  The first line is a name for our engine (this can be anything),
+  the second line is the path to the executable,
+  the third line is the flags passed to the executable,
+  and the fourth line is the initial commands passed to gtp
+  (in this case giving the bot infinite time to make moves).
+4. After adding a config, you can follow the instructions at https://youtu.be/6ZA_saVHyTA to play against newly configured engine.
+
+## More sample bot configs
+Some of the following are for internal use.
+```
+# Strongest cyclic-adversary
+cyclic-adv-s545m-v600-vm-cp505-v1-s
+/Users/ttw/code/go_attack/engines/KataGo-custom/cpp/katago
+gtp -model /Users/ttw/code/go_attack/sabaki/models/adv/cyclic-adv-s545065216-d136760487.bin.gz -victim-model /Users/ttw/code/go_attack/sabaki/models/victims/kata1-b40c256-s11840935168-d2898845681.bin.gz -config /Users/ttw/code/go_attack/configs/gtp/gtp-adv600-vm1-s.cfg
+time_settings 0 1 0
+
+# Weaker cyclic-adversary (with dragonslayer strategy)
+cyclic-adv-s349m-v600-vm-cp505-v1-s
+/Users/ttw/code/go_attack/engines/KataGo-custom/cpp/katago
+gtp -model /Users/ttw/code/go_attack/sabaki/models/adv/cyclic-adv-s349284096-d87808728.bin.gz -victim-model /Users/ttw/code/go_attack/sabaki/models/victims/kata1-b40c256-s11840935168-d2898845681.bin.gz -config /Users/ttw/code/go_attack/configs/gtp/gtp-adv600-vm1-s.cfg
+time_settings 0 1 0
+
+# Pass-trick adversary
+pass-adv-s34m-v600-vs-cp505-v1-s
+/Users/ttw/code/go_attack/engines/KataGo-custom/cpp/katago
+gtp -model /Users/ttw/code/go_attack/sabaki/models/adv/pass-adv-s34090496-d8262123.bin.gz -victim-model /Users/ttw/code/go_attack/sabaki/models/victims/kata1-b40c256-s11840935168-d2898845681.bin.gz -config /Users/ttw/code/go_attack/configs/gtp/gtp-adv600-vm1-s.cfg 
+time_settings 0 1 0
+
+# Latest with 128 visits
+cp505-v128
+/Users/ttw/code/go_attack/engines/KataGo-raw/cpp/katago
+gtp -model /Users/ttw/code/go_attack/sabaki/models/victims/kata1-b40c256-s11840935168-d2898845681.bin.gz -config /Users/ttw/code/go_attack/configs/gtp/gtp-v128.cfg
+time_settings 0 1 0
+
+# Latest with no search
+cp505-v1
+/Users/ttw/code/go_attack/engines/KataGo-raw/cpp/katago
+gtp -model /Users/ttw/code/go_attack/sabaki/models/victims/kata1-b40c256-s11840935168-d2898845681.bin.gz -config /Users/ttw/code/go_attack/configs/gtp/gtp-v1.cfg
+time_settings 0 1 0
+
+# Cyclic-adversary over ssh
+ssh-adv505h-s545m-v600-vm-cp505-v1-s
+ssh
+rnn -tt 'bash -l -c "/nas/ucb/tony/go-attack/gtp-host/go_attack/sabaki/scripts/docker-katago-custom.sh gtp -model /models/adv/cyclic-adv-s545065216-d136760487.bin.gz -victim-model /models/victims/kata1-b40c256-s11840935168-d2898845681.bin.gz -config /go_attack/configs/gtp/gtp-adv600-vm1-s.cfg"'
+time_settings 0 1 0
+```
+
+Also see https://github.com/AlignmentResearch/KataGoVisualizer/blob/3216f9b5bebaf1377dc1e320294abcc0f65bd8bd/notebooks/notebooks/paper/estimate-flops-adv.ipynb for locations of adversaries on the nas.
```

Comment:
this line will be confusing to people besides us, should we just remove it?

Reply:
Moved to separate section marked as internal